packages/notifi-react/lib/context/NotifiTopicContext.tsx

```diff
@@ -30,16 +29,18 @@ import {
 import { useNotifiFrontendClientContext } from './NotifiFrontendClientContext';
 import { useNotifiTenantConfigContext } from './NotifiTenantConfigContext';
 
+export type TopicWithFilterOption = {
+  topic: FusionEventTopic;
+  filterOptions: FusionFilterOptions;
+  subscriptionValue?: string;
+  customAlertName?: string;
+};
+
```

Comment:
Ah ...... Shall we merged MVP-6153. And these can be imported instead of use the redundant code.